air/src/options.rs

```diff
@@ -301,6 +362,42 @@ impl Deserializable for FieldExtension {
     }
 }
 
+// PARTITION OPTION IMPLEMENTATION
+// ================================================================================================
+
+/// Defines the parameters used when committing to the traces generated during the protocol.
+pub struct PartitionOption {
+    num_partitions: u8,
+    min_partition_size: u8,
+}
+
+impl PartitionOption {
+    /// Returns a new instance of `[PartitionOption]`.
+    pub fn new(num_partitions: usize, min_partition_size: usize) -> Self {
+        Self {
+            num_partitions: num_partitions as u8,
+            min_partition_size: min_partition_size as u8,
+        }
+    }
```

Comment:
Since the constructor is public, I'd add the bound checks for the inputs (we can probably just move them here).